tsconfig.json

```diff
@@ -8,7 +8,7 @@
     ],
     "types": ["react"],
     "allowJs": false,
-    "skipLibCheck": false,
+    "skipLibCheck": true,
```

Comment:
@hookform/resolvers causes typescript compilation error because of difference between yup type definition and hookform/resolvers's one